<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="API Tests"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.057">
    <properties></properties>

            <testcase
                name="createTaskSuccess"
                time="0.057"
                classname="api.Task.CreateTaskAPI">

            <failure type="java.lang.NumberFormatException" message="For input string: &quot;false&quot;">
                <![CDATA[java.lang.NumberFormatException: For input string: "false"
                java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)java.base/java.lang.Integer.parseInt(Integer.java:588)java.base/java.lang.Integer.parseInt(Integer.java:685)java_lang_Integer$parseInt$0.call(Unknown Source)org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)io.restassured.internal.common.path.ObjectConverter.convertObjectTo(ObjectConverter.groovy:29)io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)api.Task.CreateTaskAPI.createTaskSuccess(CreateTaskAPI.java:80)java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)java.base/java.lang.reflect.Method.invoke(Method.java:580)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)java.base/java.util.ArrayList.forEach(ArrayList.java:1597)org.testng.TestRunner.privateRun(TestRunner.java:739)org.testng.TestRunner.run(TestRunner.java:614)org.testng.SuiteRunner.runTest(SuiteRunner.java:421)org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)java.base/java.lang.Thread.run(Thread.java:1570)
]]>
            </failure>
    </testcase>


</testsuite>
